config/branding.php

```diff
@@ -64,6 +64,11 @@
             'icon' => 'fab fa-twitter',
             'handle' => '@' . env('SOCIAL_TWITTER', 'HSNOTTS'),
         ],
+        'mastodon' => [
+            'link' => env('SOCIAL_MASTODON', 'https://hachyderm.io/@nottinghack'),
+            'icon' => 'fab fa-mastodon',
+            'handle' => '@nottinghack@hachyderm.io',
+        ],
```

Comment:
```suggestion
        'mastodon' => [
            'link' => env('SOCIAL_MASTODON_USERNAME', 'nottinghack') ? 'https://' . env('SOCIAL_MASTODON_DOMAIN', 'hachyderm.io') . '/@' . env('SOCIAL_MASTODON_USERNAME', 'nottinghack') : null,
            'icon' => 'fab fa-mastodon',
            'handle' =>'@' . env('SOCIAL_MASTODON_USERNAME', 'nottinghack') . '@' . env('SOCIAL_MASTODON_DOMAIN', 'hachyderm.io'),,
        ],
```

need to build this in two parts and allow that is in `.env` someone has "SOCIAL_MASTODON_USERNAME=" bank then it won't show

with the above you can add 
```
        if (config('branding.social_networks.mastodon.link')) {
            $spaceApi['contact']['mastodon'] = config('branding.social_networks.mastodon.handle');
        }
        
```
into `SpaceApiController` above the line `if (config('branding.theme') == 'nottinghack') {`